pallets/liquidity-pools-gateway/routers/src/lib.rs

```diff
@@ -24,6 +24,9 @@ pub const DEFAULT_PROOF_SIZE: u64 = 256 * 1024;
 // See moonbeam docs: https://docs.moonbeam.network/builders/interoperability/xcm/fees/#:~:text=As%20previously%20mentioned%2C%20Polkadot%20currently,1%2C000%2C000%2C000%20weight%20units%20per%20instruction
 pub const XCM_INSTRUCTION_WEIGHT: u64 = 1_000_000_000;
 
+/// Multiplier for converting a unit of gas into a unit of Substrate weight
+pub const GAS_TO_WEIGHT_MULTIPLIER: u64 = 25_000;
```

Comment:
May I ask how this amount was derived?

Reply:
Moonbeam. We have a maximum block weight of '500_000_000_000'.  Moonbeam uses 75% of this for EVM logic. We go with the same. They want the gas limit of the EVM to be the same as Ethereum (`15_000_000`). 

This gives us: `375_000_000_000 / 15_000_000 = 25_000`.

Reply:
Thanks for the thorough explanation 🫶